mvpa2/algorithms/group_clusterthr.py

```diff
@@ -174,18 +174,40 @@ class GroupClusterThreshold(Learner):
             will vary across features yielding a threshold vector. The number
             of bootstrap samples need to be adequate for a desired probability.
             A ``ValueError`` is raised otherwise.""")
+    # Note that  feature_thresh_prob  is relevant only for "cluster forming" so
+    # will be irrelevant for e.g. metric="max_value"
+    # TODO: address in documentation or some RF to make "Cluster*" implementation
+    # specific (e.g. a subclass?)
 
     chunk_attr = Parameter(
-        'chunks',
+        'chunks', constraints=EnsureStr() | EnsureNone(),
         doc="""Name of the attribute indicating the individual chunks from
             which a single sample each is drawn for averaging into a bootstrap
-            sample.""")
+            sample.  If None, no bootstrapping will be performed, and each
+            sample will be used as a NULL result sample, thus there should be
+            a sufficient number of them""")
+    # TODO: note that n_bootstrap will not be relevant if chunks_attr=None
+    #       Address either in documentation or by subclassing...
 
     fwe_rate = Parameter(
         0.05, constraints=EnsureFloat() & EnsureRange(min=0.0, max=1.0),
         doc="""Family-wise error rate for multiple comparison correction
             of cluster size probabilities.""")
 
+    metric = Parameter(
+        'cluster_sizes',
+        constraints=EnsureChoice('max_cluster_size', 'cluster_sizes', 'cluster_sizes_non0',
+                                 'max_value'),
+        doc="""What metric is measured per each value to be aggregated into
+            H0 distribution.  'max_cluster_size' collects a distribution of
+            maximal cluster size detected in a sample (or 0), 'cluster_sizes'
+            collects a distribution of all cluster sizes encountered while providing
+            a single 0 value per map if no clusters in that map were detected.
+            'cluster_sizes_non0' does not count 0s. 'max_value' collects a distribution
+            of maximal values.
+            Some metrics ('max_cluster_size', 'max_value') estimate family-wise
+            metric so no post-hoc correction is strictly necessary.""")
+
```

Comment:
What does max_value do? This happens after thresholding, so if only max value is of interest, there is no need for thresholding at all.